src/js/control-bar/time-controls/current-time-display.js

```diff
@@ -55,7 +55,10 @@ class CurrentTimeDisplay extends Component {
     let time = (this.player_.scrubbing()) ? this.player_.getCache().currentTime : this.player_.currentTime();
     let localizedText = this.localize('Current Time');
     let formattedTime = formatTime(time, this.player_.duration());
-    this.contentEl_.innerHTML = `<span class="vjs-control-text">${localizedText}</span> ${formattedTime}`;
+    if(formattedTime !== this.formattedTime_){
```

Comment:
needs whitespace after `if` and before `{`, in `remaining-time-display` as well.